package.json

```diff
@@ -16,6 +16,7 @@
         "antd": "^4.14.1",
         "bignumber.js": "^9.0.1",
         "bnc-onboard": "^1.30.0",
+        "body-scroll-lock": "^4.0.0-beta.0",
```

Comment:
what does this package provide?

Reply:
It prevents the user from scrolling the page and the elements jumping when the Tour is open